### DISCUSSION TITLE
Coursier and brew on macOS

### DISCUSSION THREAD

**Discussion by halloleo** [coursier/coursier#3023] [Q&A] [answered]
I have installed `scala`, `scala-cli` and `coursier` on my MacBook via homebrew. Now I ran `coursier setup` and it downloaded  `scala`, `scala-cli` again into its strange bin location at `~/Library/Application Support/Coursier/bin`.

Two questions:

1. How can I tell coursier to recognize the already brew-installed  `scala`, `scala-cli` commands and _not_ to download them again?
2. How can I tell coursier to save further binaries to a more unix-y location like `~.local/bin` (and to use for the cache `~/.cache/coursier/v1`?

Any pointers are much appreciated!

PS: Homebrew's  `scala`, `scala-cli` were already in $PATH _before_ I ran `coursier setup`, so coursier should have found them already.

**Comment by halloleo** [comment]
Actually I found answers to point 2:

Set the environment variables `$COURSIER_BIN_DIR` and `$COURSIER_CACHE` _before_ running coursier.

Still unclear how to make `coursier setup` recognise what commands are already in my $PATH.

**Reply by tgodzik** [reply, marked as answer]
I don't think that's possible currently, you would need to just install each component separately.
